tests/flow/dictionary/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -43,9 +43,10 @@ function foo0(
   return x;
 }
 
-function foo2(
-  x: { [key: string]: number }
-): { [key: string]: number, +toString: () => string } {
+function foo2(x: { [key: string]: number }): {
+  [key: string]: number,
+  +toString: () => string
+} {
```

Comment:
Can you add
```es6
function foo2(x: {
  [key: string]: number,
  toString: () => string,
}): { [key: string]: number } {}
```
and
```es6
function foo2(x: {
  [key: string]: number,
}): { [key: string]: number } {}
```
to the tests? The return type should not be split over multiple lines if it has only on key in the type.